person_class.rb

```diff
@@ -0,0 +1,30 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(parent_permission, name = 'Unknown', age: true)
+    @id = Random.rand(1...1000)
+    @parent_permission = parent_permission
+    @name = name
+    @age = age
+  end
+
+  # name setter
+  def setname=(name)
+    @name = name
+  end
+
+  # age setter
+  def setage=(age)
+    @age = age
+  end
+
```

Comment:
[OPTIONAL]
- [ ] `attr_accessor` among [others](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)  are shortcuts to write GETTERS and SETTERS. Since you already used that, you do not need to write again the plain GETTER and SETTER methods here. Kindly feel free to remove these.

Reply:
It was a good feedback! Thank you!